daemon/src/process_manager.rs

```diff
@@ -119,8 +124,22 @@ impl Actor {
                     })
                     .await?;
             }
-            OracleAttestedPostCetTimelock { cet, .. }
-            | CetTimelockExpiredPostOracleAttestation { cet } => {
+            CetTimelockExpiredPostOracleAttestation { cet } => {
+                self.try_broadcast_transaction
+                    .send_async_safe(monitor::TryBroadcastTransaction {
+                        tx: cet,
+                        kind: TransactionKind::Cet,
+                    })
+                    .await?;
+            }
+            OracleAttestedPostCetTimelock { cet, .. } => {
+                let _ = self
+                    .monitor_cet_finality
+                    .send(MonitorCetFinality {
```

Comment:
Please use `send_async_safe` here. For why, please see https://github.com/itchysats/itchysats/pull/1200, especially the last commit.